python/sklearnserver/README.md

```diff
@@ -0,0 +1,71 @@
+# Scikit-Learn Server
+
+[Scikit-Learn](https://scikit-learn.org/stable/) server is an implementation of KFServing for serving Scikit-learn models, and provides a Scikit-learn model implementation for prediction, pre and post processing. 
+
+To start the server locally for development needs, run the following command under this folder in your github repository. Also please ensure you have installed the [kfserving](../kfserving) before.
+
+```
+pip install -e .
```

Comment:
This should actually install kfserving since it's a dependency, so your above comment shouldn't be necessary

Reply:
Done